CourseApp/Program/DateClass.cs

```diff
@@ -0,0 +1,89 @@
+namespace CourseApp.Program
+{
+    using System;
+    using static System.Math;
+
+    public class DateClass
+    {
+        public (int, int, int) CalculateAge(DateTime birthDay, DateTime dateToday)
```

Comment:
Let's create struct (or class) that will hold these 3 values in human readable format - Year, Month, Day